pycvvdp/display_model.py

```diff
@@ -308,6 +331,11 @@ def forward( self, V ):
             L = pq2lin( V ).clip(0.005, self.Y_peak) + Y_black + Y_refl #TODO: soft clipping
         elif self.EOTF=='linear':
             L = V.clip(max(0.005, Y_black), self.Y_peak) + Y_refl #TODO: soft clipping
+        elif self.EOTF=='HLG':
```

Comment:
I believe 'HLG' should also be added to the exception from clipping (at the line 322), as values below 0 and above 1 are allowed (see note 5g in the recommendation).